src/core_modules/capture-core/rules/converters/outputConverter.js

```diff
@@ -1,68 +1,106 @@
 // @flow
 /* eslint-disable class-methods-use-this */
+import log from 'loglevel';
 import moment from 'moment';
-import { convertMomentToDateFormatString } from '../../../utils/converters/date';
-import type { IConvertOutputRulesEffectsValue } from '../rulesEngine.types';
+import type { IConvertOutputRulesEffectsValue } from 'capture-core-utils/rulesEngine/rulesEngine.types';
+import { convertMomentToDateFormatString } from '../../utils/converters/date';
 
 const dateMomentFormat = 'YYYY-MM-DD';
 
 class RulesValueConverter implements IConvertOutputRulesEffectsValue {
     convertText(value: string): string {
         return value;
     }
+
     convertLongText(value: string): string {
         return value;
     }
+
     convertLetter(value: string): string {
         return value;
     }
+
     convertPhoneNumber(value: string): string {
         return value;
     }
+
     convertEmail(value: string): string {
         return value;
     }
+
     convertBoolean(value: boolean): string {
         return value ? 'true' : 'false';
     }
+
     convertTrueOnly(): string {
         return 'true';
     }
+
     convertDate(value: string): string {
         const momentDate = moment(value, dateMomentFormat);
         return convertMomentToDateFormatString(momentDate);
     }
+
     convertDateTime(value: string): ?Object {
         const momentDateTime = moment(value);
         return {
             date: convertMomentToDateFormatString(momentDateTime),
             time: momentDateTime.format('HH:mm'),
         };
     }
+
     convertTime(value: string): string {
         return value;
     }
+
     convertNumber(value: number): string {
         return value.toString();
     }
+
+    convertUnitInterval(value: number): string {
+        return value.toString();
+    }
+
+    convertPercentage(value: number): string {
+        return value.toString();
+    }
+
     convertInteger(value: number): string {
         return value.toString();
     }
+
     convertIntegerPositive(value: number): string {
         return value.toString();
     }
+
     convertIntegerNegative(value: number): string {
         return value.toString();
     }
+
     convertIntegerZeroOrPositive(value: number): string {
         return value.toString();
     }
-    convertPercentage(value: number): string {
-        return value.toString();
+
+    convertTrackerAssociate(value: string): any {
+        log.warn('convertTrackerAssociate not implemented', value);
```

Comment:
hey @superskip, is this something we will implement in the future or what is the plan?

Reply:
That particular function? Probably not from what I've heard, but the function needs a definition because of the interface that RulesValueConverter implements. The interface is defined such that each _value type_ a data element can have has one converter, and _Tracker Associate_ is one such value type.